### ISSUE TITLE
Can't submit Application service after created a new tier

### ISSUE THREAD

**Issue by pek77** [awslabs/aws-saas-boost#519]
### The Question
Can't submit Application service after created a new tier.

I have created a new tier.

But when I submit Application service with default tier or new tier.

It shown a Error : `Errors in form: Service main Configurations compute Tiers [tier1]`

I just follow the guide after create a tier: https://github.com/awslabs/aws-saas-boost/blob/main/docs/getting-started.md#step-5---configure-tiers-and-application-settings

Maybe it is a bug?

### Environment

  - **AWS SaaS Boost Version:** {"tag":"v2.2.3","commit":"213eccb","describe":"v2.2.3-1"}

### Other information

![图片](https://github.com/awslabs/aws-saas-boost/assets/12756666/87a36a1f-9a2d-4af9-b8c6-6bade4ee3c71)

**Comment by brtrvn** [comment]
Under tier settings have you selected the compute choices for each tier (even if the green check mark is decorating the input widget)?

**Comment by pek77** [comment]
> Under tier settings have you selected the compute choices for each tier (even if the green check mark is decorating the input widget)?

Yes, I have posted a GIF FYI (about 2min30s ), it looks like a UI Bug.

![error](https://github.com/awslabs/aws-saas-boost/assets/12756666/a96e3d89-a472-492f-82a5-77db99c41521)
